### DISCUSSION TITLE
求助：如何进入绳网？

### DISCUSSION THREAD

**Discussion by anything12121** [share121/inter-knot#2319] [其他]
试了好多次，也无法下载脚本，无法打开

**Comment by guozichang2007** [comment]
教程:1.先下载Steam++，浏览器接搜就行，选Github加速。
2.打开视频里github.com/share121/inter-knot，然后注册登录Github用户，登录完会跳到自己的页面，再输一次github.com/share121/inter-knot就行，网页右边有一个绳网，下面是链接，打开然后它会提醒你安装脚本，点进去以后不要动。
3.浏览器搜索tampermonkey，然后找到添加这个浏览器插件就行，然后回到绳网助手页面，点击安装，后面油猴就会跳出来直接安装就行。
4.回到绳网，应该就进去了

**Comment by Aha-SWYH** [comment]
【🔥PWA 毁了我的绳网🔥-哔哩哔哩】 https://b23.tv/qcWN2jw 进这个看看

**Comment by LuAn-afl** [comment]
纳尼？这是在玩什么抽象吗？你不会那你怎么发的帖子

**Reply by Aha-SWYH** [reply]
不是吧，毕竟能进页面加载不出来 确实能写帖子

**Reply by LuAn-afl** [reply]
> 不是吧，毕竟能进页面加载不出来 确实能写帖子

可是楼主是直接无法下载脚本唉如果下不了脚本就不会出现在绳网了吧

**Reply by opqphp** [reply]
可以在GitHub直接发的  但没绳网这个页面

**Reply by kaijinti** [reply]
确实可以不进来就发帖，之前我也是，点进绳网什么都没有，但是发帖的那个按钮可以点，我当时也发了一个为什么不能用了的帖子
